modules/bitgo/src/v2/coins/near.ts

```diff
@@ -166,15 +170,14 @@ export class Near extends BaseCoin {
    * Explain/parse transaction
    * @param params
    */
-  async explainTransaction(
-    params: ExplainTransactionOptions,
-  ): Promise<NearTransactionExplanation> {
+  async explainTransaction(params: ExplainTransactionOptions): Promise<NearTransactionExplanation> {
     const factory = accountLib.register(this.getChain(), accountLib.Near.TransactionBuilderFactory);
     let rebuiltTransaction: accountLib.BaseCoin.BaseTransaction;
-    const txRaw = params.txPrebuild.txHex;
+    const txHex = params.txPrebuild.txHex;
+    const rawTxBase64 = HEX_REGEX.test(txHex) ? Buffer.from(txHex, 'hex').toString('base64') : txHex;
 
     try {
-      const transactionBuilder = factory.from(txRaw);
+      const transactionBuilder = factory.from(rawTxBase64);
       rebuiltTransaction = await transactionBuilder.build();
```

Comment:
add a comment 
// TODO(STLX-16118) Change parse responsability to the from() method in Account-lib

Reply:
Done